src/Kunstmaan/NodeBundle/Entity/PageTabInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Kunstmaan\NodeBundle\Entity;
+
+use Kunstmaan\NodeBundle\ValueObject\PageTab;
+
+interface PageTabInterface {
```

Comment:
Code style for this repo requires newlines before opening class/interface/trait braces